common-custom-user-data-gradle-plugin/README.md

```diff
@@ -15,6 +15,9 @@ Out of the box, this plugin enhances all build scans published with additional t
 - A tag representing builds run on CI, together with a set of tags, links and custom values specific to the CI server running the build.
 - For `git` repositories, information on the commit id, branch name, status, and whether the checkout is dirty or not.
 
+Out of the box, this plugin also allows users to override various Gradle Enterprise, build scan, and build caching settings via
```

Comment:
You could also add a markdown link to the documentation for this here.

Reply:
Good idea. Done!